climpred/stats.py

```diff
@@ -228,6 +228,7 @@ def xr_rm_trend(da, dim='time'):
     return xr_rm_poly(da, 1, dim=dim)
 
 
+# # TODO: coords lon, lat get lost for curvilinear ds
```

Comment:
I think we should be able to fix this at some point with the `xarray` config options, to keep attrs and coords.